src/lib/utils.js

```diff
@@ -1,16 +1,32 @@
 import ck from 'creditkey-js';
 
 const publicKeys = {
-  development: 'thewebstaurantstoreinc_1a3fc138f571474b997815ab57febbfe',
+  development: 'creditkeydev_2822baea77774929979f1e5964dd18b6',
   staging: 'creditkeytest_6d8e5758033846b4995993dd74dda57c',
   production: 'creditkeysamplestore_2d3170af046347cdaf0171e06c08ca77'
 }
 
+const pi4PublicKeys = {
+  development: 'thewebstaurantstoreinc_dcf536bfabb748999f1bf57dda2630c6', // eben local
```

Comment:
it would be very useful to be able to define these keys in the `.env` files  😄

Reply:
agreed